bin/dsn.cmake

```diff
@@ -231,7 +231,7 @@ endmacro(ms_replace_compiler_flags)
 function(ms_check_cxx11_support)
     if(UNIX)
         include(CheckCXXCompilerFlag)        
-        CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+        CHECK_CXX_COMPILER_FLAG("-std=c++1y" COMPILER_SUPPORTS_CXX11)
```

Comment:
let's stick to C++11 to avoid asking developers too much about compilers. Unless you have something that is very important and cannot be implemented using C++11 and below.